tests/codegen_testcases/solidity/unused_variable_elimination.sol

```diff
@@ -115,42 +122,42 @@ return 3;
         it2 = 1;
 
         return 2;
-// CHECK: store storage slot((overflowing %t2 + uint256 0)) ty:int256 =
-// CHECK: store storage slot(uint256 2) ty:int256 =
-// NOT-CHECK: ty:struct c1.testStruct %t3 = struct { int256 1, int256 2 }
-// NOT-CHECK: alloc int256[] len uint32 5
-// NOT-CHECK: store storage slot(uint256 3) ty:int256
+        // CHECK: store storage slot((overflowing %t2 + uint256 0)) ty:int256 =
+        // CHECK: store storage slot(uint256 2) ty:int256 =
+        // NOT-CHECK: ty:struct c1.testStruct %t3 = struct { int256 1, int256 2 }
+        // NOT-CHECK: alloc int256[] len uint32 5
+        // NOT-CHECK: store storage slot(uint256 3) ty:int256
     }
 
-// BEGIN-CHECK: c3::function::test9
+    // BEGIN-CHECK: c3::function::test9
     function test9() public view returns (int) {
         int f = 4;
 
-        int c = 32 +4 *(f = it1+it2);
-//CHECK: ty:int256 %f =
+        int c = 32 + 4 * (f = it1 + it2);
+        //CHECK: ty:int256 %f =
         return f;
     }
 
-// BEGIN-CHECK: c3::function::test10
+    // BEGIN-CHECK: c3::function::test10
     function test10() public view returns (int) {
         int f = 4;
 
-        int c = 32 +4 *(f = it1+it2);
-// CHECK: ty:int256 %c = (int256 32 + (sext int256 (int64 4 * (trunc int64 (%temp.126 + %temp.127)))))
-// NOT-CHECK: ty:int256 %f = (%temp.10 + %temp.11)
+        int c = 32 + 4 * (f = it1 + it2);
+        // CHECK: ty:int256 %c = (int256 32 + (sext int256 (int64 4 * (trunc int64 (%temp.130 + %temp.131)))))
+        // NOT-CHECK: ty:int256 %f = (%temp. + %temp.)
```

Comment:
Yeah good point. I mean this was very subtle before too. What do you think would be a better way for writing the NOT-CHECK so that temp var numbers don't play any role?

Reply:
I just removed the numbers so it just checks for the `f` var assignment